### DISCUSSION TITLE
How to generate package for windows app ?

### DISCUSSION THREAD

**Discussion by RonakPatelDotNet** [dotnet/maui#4182] [Q&A]
Visual studio 2022 not opening anything when i click project and select 'publish'. (no error prompt too)
I have no idea how to generate package for windows. I want to generate package so my other team member can install app & test it.

**Comment by Xyncgas** [comment]
I can publish stuffs, I can't understand how to publish (/use the published stuffs) my app though, is it something for users to install or is it something for windows store to distribute

**Reply by Xyncgas** [reply]
I'd like to just let people download my app and install it without going through the windows store, so I can make whatever I want without giving money to the store, I can just give people a link to my website and they can download and use my app

**Reply by Xyncgas** [reply]
Which I can already do with electron
